pkg/skaffold/deploy/kustomize.go

```diff
@@ -97,15 +98,10 @@ func (k *KustomizeDeployer) Labels() map[string]string {
 
 // Deploy runs `kubectl apply` on the manifest generated by kustomize.
 func (k *KustomizeDeployer) Deploy(ctx context.Context, out io.Writer, builds []build.Artifact, labellers []Labeller) *Result {
-	if err := k.kubectl.CheckVersion(ctx); err != nil {
-		color.Default.Fprintln(out, "kubectl client version:", k.kubectl.Version(ctx))
-		color.Default.Fprintln(out, err)
-	}
-
-	manifests, err := k.readManifests(ctx)
+	manifests, err := k.renderManifests(ctx, out, builds, labellers)
```

Comment:
I think there should be a `event.DeployInProgress()` here.

Reply:
You're right. Added!